stdlib/public/core/Character.swift

```diff
@@ -398,8 +398,7 @@ extension String {
       self = String._fromWellFormedCodeUnitSequence(
         UTF8.self, input: smallUTF8)
     case let .large(value):
-      let buf = String(_StringCore(_StringBuffer(value)))
-      self = buf[buf.startIndex..<buf.index(after: buf.startIndex)]
+      self = String(_StringCore(_StringBuffer(value)))
```

Comment:
...which explains why the tests failed.  Clearly we can't accept the PR as-is, but it was not meant to be a candidate for merging; just a demonstration of possible speedup.  The question is, what are we gaining by putting off finding the end of the first grapheme?  I don't know.  If there's time for me to land my Character rewrite, we'll find out which is faster eventually anyhow, because it does not delay like this.